server/util.go

```diff
@@ -280,26 +280,26 @@ func interceptConfigs(rootViper *viper.Viper, customAppTemplate string, customCo
 
 // add server commands
 func AddCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator types.AppCreator, appExport types.AppExporter, addStartFlags types.ModuleInitFlags) {
-	tendermintCmd := &cobra.Command{
+	cometCmd := &cobra.Command{
 		Use:   "tendermint",
```

Comment:
are we not going to change this usage?

Reply:
Not in this PR, as this one should be totally non-breaking. I want to discuss in the standup to which point we want to be breaking. Anyway, I can already add an alias here.